From 572b6191b5705f6e98e84dd7ff1fff91efca024b Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Thu, 2 Nov 2017 10:55:32 -0700
Subject: [PATCH] DOCKER-1123 docker build v1 image test should cleanup it's
 test container Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent
 Mick <trentm@gmail.com>

---
 test/integration/api-images-v1-v2.test.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/integration/api-images-v1-v2.test.js b/test/integration/api-images-v1-v2.test.js
index 119673c..2cbbe72 100644
--- a/test/integration/api-images-v1-v2.test.js
+++ b/test/integration/api-images-v1-v2.test.js
@@ -388,7 +388,7 @@ test('test for error on docker v1 image build', function (tt) {
                 h.buildDockerContainer({
                     dockerClient: DOCKER_ALICE_HTTP,
                     params: {
-                        'rm': 'true'  // Remove container after it's built.
+                        'forcerm': 'true'  // Remove container after it's built.
                     },
                     test: t,
                     tarball: ctx.tarStream
-- 
2.21.0

